addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3527,8 +3527,10 @@ export class OdooEditor extends EventTarget {
                     const editorTabs = new Set(
                         [...nonListItems].map(node => {
                             const block = closestBlock(node);
-                            if (block && isEditorTab(block.firstElementChild)) {
-                                return block.firstElementChild;
+                            for(const child of descendants(block)) {
+                                if(isEditorTab(child)) {
+                                    return child;
```

Comment:
@adsa-odoo Hi, when you don't need to keep track of the index or perform complex looping conditions. prefer using `foreach` 
`
descendants(block).forEach(child => {
    if (isEditorTab(child)) {
        return child;
    }
});
`
use this instead.

Reply:
Okay! Got it.

Reply:
@adsa-odoo my bad The return statement does not work within a forEach loop. The forEach method is designed for iterating over elements of an array and executing a callback function for each element. However, it does not provide a mechanism to return a value from the loop itself.

If you need to return a specific value from the loop, you might consider using a for...of loop instead. So your logic will work.